syntax = "proto3";

package  payment;

service PaymentService {
    rpc CreatePayment(PaymentRequest) returns (PaymentResponse);
}